script/utils/getPageUrlOrRedirection.ts

```diff
@@ -0,0 +1,7 @@
+import { Redirects } from '../../data/redirects/types';
+
+const getPageUrlOrRedirection = (redirects: Redirects, page: string) => {
+    return redirects.find(({ from }) => from === page) || page;
```

Comment:
it is returning `Redirect | string`, but should always return `string`
Also, I think you should compare `page` to `redirect.to` and return `redirect.from` in order to check if it is working because it is checking on `openstax.org` that doesn't have a new version yet